stardis/radiation_field/opacities/opacities_solvers/base.py

```diff
@@ -373,6 +373,8 @@ def calc_alpha_line_at_nu(
     if line_opacity_config.disable:
         return 0, 0, 0
 
+    no_depth_points = len(stellar_model.geometry.r)
```

Comment:
`no_of_depth_points` may be clearer. Otherwise this may be read as "no depth points"

Reply:
Maybe `depth_point_count`?